packages/common/pipes/file/parse-file.pipe.ts

```diff
@@ -39,7 +39,7 @@ export class ParseFilePipe implements PipeTransform<any> {
   }
 
   async transform(value: any): Promise<any> {
-    if (isUndefined(value)) {
+    if (isUndefined(value) || (Array.isArray(value) && isEmpty(value))) {
```

Comment:
This `if` statement started to grow its logic here. Could you wrap this into a private method and give it a meaningful name? something like:

```suggestion
    if (this.thereAreNoFilesIn(value)) {
```